src/Symfony/Component/Form/AbstractType.php

```diff
@@ -64,8 +64,16 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
+    public function getName()
+    {
+        return;
```

Comment:
Interesting. This is a clear violation of our [BC policy](http://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces).